topology/Jackson State University/JSU RC/JSU-RC.yaml

```
@@ -0,0 +1,41 @@
+Production: true
+SupportCenter: Self Supported
+GroupDescription: This is an OSPool Access Point on our RC Cluster
+
+Resources:
+  JSU-ap0:
+    Active: false
```

```suggestion
    Active: true
```

Otherwise you won't be able to fetch a token